### ISSUE TITLE
Would like to have a preferred style preference, or view=standard as default style

### ISSUE THREAD

**Issue by pkoppenb** [indico/indico#4894]
**Is your feature request related to a problem? Please describe.**
At each indico meeting that uses the default timetable view, I always manually switch to "indico style". Based on what I see in shared zoom screens, most users seem to do that. I thus wonder why it's not the default.

**Describe the solution you'd like**
Either (or both)
1) set indico style to default
2) allow me to save that in my preferences so that https://indico.cern.ch/event/1016060/timetable is effectively https://indico.cern.ch/event/1016060/timetable/?view=standard

**Describe alternatives you've considered**
I looked if there was a hidden setting but couldn't find any.

**Comment by pferreir** [comment]
My understanding is that many people do that because they want to have a sequential view of the event. In events such as the one you link to, it is indeed more useful than the default "conference" view, which was thought for events with parallel sessions. If we managed to provide a view of the timetable which is more useful, my guess is that far less people would switch.

We are planning to rework the conference timetable rendering code at some point in 2022, maybe this is something to take into account in the requirements for that development, rather than resorting to hacks. /cc @ThiefMaster

**Comment by ThiefMaster** [comment]
Yeah, I think with the current system it doesn't make that much sense - you'd lose the menu etc.

Providing a sequential timetable view within the conference view in the future sounds indeed like a much better option than trying to add a setting like the one you're asking for right now..
